src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/Streaming/DataStreamReaderTests.cs

```diff
@@ -30,6 +31,12 @@ public void TestSignaturesV2_3_X()
 
             Assert.IsType<DataStreamReader>(dsr.Format("parquet"));
 
+            Assert.IsType<DataStreamReader>(
+                dsr.Schema(
+                    new StructType(new[]
+                    {
+                        new StructField("columnName", new IntegerType())
+                    })));
```

Comment:
I think this can fit in one line.

Reply:
I am following the format in [dictionary](https://github.com/elvaliuliuliu/spark/blob/37be26f8fe816ea9556bd53247aa3e8f8b755562/src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/Streaming/DataStreamReaderTests.cs#L47)

I think we can either change both to one line or keep separate to be consistent. Thanks.

Reply:
Sure you can keep it the way it is.

Reply:
Cool, sounds good.